vm/src/obj/objtype.rs

```diff
@@ -331,6 +331,7 @@ fn linearise_mro(mut bases: Vec<Vec<PyClassRef>>) -> Option<Vec<PyClassRef>> {
     Some(result)
 }
 
+#[allow(clippy::implicit_hasher)]
```

Comment:
We could probably disable this one at the crate level, I don't think it's useful at all